packages/celer-web-app/src/core/engine/RouteDocLine.ts

```diff
@@ -33,6 +33,7 @@ export interface DocLineText {
     notes?: TypedString,
 
     centerCoord?: InGameCoordinates;
+    highlighted: boolean;
```

Comment:
I do not recommend doing this. The engine is configured to only re-compute the doc if the content needs to change. I don't see a change to re-run the engine when highlighted line changes, but I assume that's what you want to do. The engine does take some settings as an input, but for purely visual effects, it's better to just do it in the component that consumes the engine output.

(for example, see how the components change color when you change the theme. The engine does not re-run since the doc content is the same, and only the color changes)